assembl/templates/views/navBarRight.tmpl

```diff
@@ -20,6 +21,7 @@
     </li>
     {% endif %}
     {% endif %}
+    {% endif %}
```

Comment:
Yeah, agreed, not in the scope of this PR. Just saw the code, actually. It's not that big of a deal, honestly. I just know that new devs will have a really fun time hunting template bugs if one emerges. This is not priority. And indeed is scope creep.